checkerboard_detector/src/checkerboard_detector.cpp

```diff
@@ -587,13 +595,34 @@ class CheckerboardDetector
                 tglobal.rot = Vector(vobjects[i].pose.orientation.w,vobjects[i].pose.orientation.x,vobjects[i].pose.orientation.y, vobjects[i].pose.orientation.z);
                 Transform tlocal = tglobal * cb.tlocaltrans.inverse();
 
+                // draw all the points
+                int csize0 = std::max(circle_size_, 6);
+                int cwidth0 = std::max(circle_size_/3, 2);
+                int csize1 = std::max((2*circle_size_)/3, 4);
+                int csize2 = std::max(circle_size_/4, 2);
+                int cwidth1 = std::max(circle_size_/2, 4);
+                int cwidth2 = std::max(circle_size_/2, 3);
+
+                for(size_t i = 0; i < cb.grid3d.size(); ++i) {
+                    Vector grid3d_vec(cb.grid3d[i].x, cb.grid3d[i].y, cb.grid3d[i].z);
+                    Vector p = tlocal * grid3d_vec;
+                    dReal fx = p.x*camInfoMsg.P[0] + p.y*camInfoMsg.P[1] + p.z*camInfoMsg.P[2] + camInfoMsg.P[3];
+                    dReal fy = p.x*camInfoMsg.P[4] + p.y*camInfoMsg.P[5] + p.z*camInfoMsg.P[6] + camInfoMsg.P[7];
+                    dReal fz = p.x*camInfoMsg.P[8] + p.y*camInfoMsg.P[9] + p.z*camInfoMsg.P[10] + camInfoMsg.P[11];
+                    int x = (int)(fx/fz);
+                    int y = (int)(fy/fz);
+                    cv::circle(frame, cv::Point(x,y), csize0, cv::Scalar(0, 255, 0), cwidth0);
```

Comment:
Have you changed color from (0,0,0) to (0,255,0)? Then could you please post new visualization on this PR?